src/renderer/components/+workloads-pods/pods.store.ts

```diff
@@ -32,15 +31,9 @@ export class PodsStore extends KubeObjectStore<Pod> {
     }
   }
 
-  getPodsByOwner(workload: WorkloadKubeObject): Pod[] {
```

Comment:
Breaking change

Reply:
Namely, `PodsStore` is an exported type.

Reply:
Sure thing! I will return an old function and add `getPodsByOwnerId()` as a separate one.